plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -216,7 +223,9 @@ export class MySQLInstrumentation extends InstrumentationBase<
         }
       }
       if (typeof cb === 'function') {
-        cb(...arguments);
+        context.with(activeContext, () => {
```

Comment:
can you please cover that with unit test ?

Reply:
@obecny done, please can you take another look?

Reply:
@vmarchaud pls avoid resolving comments which you are not author of - I have missed this conversation as it was resolved already :/

Reply:
@sstone1 pls don't force push after receiving first review or comment from anyone, it is impossible to see changes. The PR is stashed once it is merged anyway